routes/index.js

```diff
@@ -15,8 +15,22 @@ const GS_REQUEST_TYPE_ID = process.env.GS_REQUEST_TYPE_ID;
 const headerOptions = {
   username: JIRA_USERNAME,
   password: JIRA_KEY,
-  accept: "application/json",
-  content_type: "application/json",
+  accept: 'application/json',
+  content_type: 'application/json',
+};
+
+//build request description based on which form is being submitted
+const buildDescription = async (requestBodyObject) => {
+  if (requestBodyObject.formName == 'basic-form') {
+    return `*GENERAL FEEDBACK* \n\n Book description or URL: ${requestBodyObject.bookDescription} \n Full description: ${requestBodyObject.description} \n\n User agent: ${requestBodyObject.userAgent} \n User URL: ${requestBodyObject.userURL} \n User auth: ${requestBodyObject.userAuthStatus}`;
+  } else if (requestBodyObject.formName == 'catalog-correction') {
+    return `*CATALOG QUALITY CORRECTION* \n\n URL of catalog record: ${requestBodyObject.recordURL} \n URL of item with issue within record: ${requestBodyObject.itemURL} \n Title of book: ${requestBodyObject.itemTitle} \n Problems: ${requestBodyObject.problems} \n Access issues: ${requestBodyObject.access} \n\n Other: ${requestBodyObject.description} \n\n User agent: ${requestBodyObject.userAgent} \n User URL: ${requestBodyObject.userURL} \n User auth: ${requestBodyObject.userAuthStatus}`;
+    // return `Book description or URL: ${requestBodyObject.bookDescription} \n Full description: ${requestBodyObject.description} \n user agent: ${requestBodyObject.userAgent} \n user URL: ${requestBodyObject.userURL} \n user auth: ${requestBodyObject.userAuthStatus}`;
+  } else if (requestBodyObject.formName == 'content-correction') {
+    return `*CONTENT QUALITY CORRECTION* \n\n URL of book with problem: ${requestBodyObject.bookURL} \n Title of book: ${requestBodyObject.itemTitle} \n Overall quality: ${requestBodyObject.imageQuality} \n Specific page image problems: ${requestBodyObject.imageProblems} \n\n Other: ${requestBodyObject.description} \n\n User agent: ${requestBodyObject.userAgent} \n User URL: ${requestBodyObject.userURL} \n User auth: ${requestBodyObject.userAuthStatus}`;
+  } else {
+    return `form name issue`;
```

Comment:
Should this raise an error if the form name doesn't match one of the known ones? As is if I'm understanding this, I think it would create an issue in Jira, but where the body just said "form name issue"?

Reply:
Yeaahhhh, the message here should be more robust. I haven't run into this error in testing, but theoretically, yes, the ticket would still be created just without any real data in the "description" box in Jira. I'm trying to imagine what kind of message would even be helpful here... "There was an error in collecting additional info from the form, please contact the ticket submitter"??? (name/email/summary would all still make it through)

Maybe I should ask Val. I don't foresee this `else` condition ever actually playing out, but that doesn't mean we shouldn't have a useful message for that eventuality. 

Unless you think it should stop the form submission altogether? I could have the feedback collector send back an error to front-end to let the user know to contact support via email (that's the very last fall-back in the error chain on the front-end).

Reply:
Yes, I think it should stop the form submission altogether - the main case I could see it happening is if someone is messing with the api proxy endpoint separate from the front-end form entirely.

Reply:
Added some variables that will trigger error handling and output a message (hopefully to help with debugging) in the event that the `else` condition is met.